### PR TITLE
Force Configured Company Name option in Income Statement

### PR DESCRIPTION
**Pull request by debian-testing**
Similar to PR #1330, but for Income Statement

### CODE REVIEW

**Comment by debian-testing** [comment]
Use only the configured company name and remove the option field from the report for the Income statement report.

**Comment by fellen** [comment]
A note on your git usage:
you should never merge maint,
instead rebase your branch on maint's HEAD and
finally force-push it into your public repo.l

File: gnucash/report/reports/standard/income-statement.scm [hunk 1]
```diff
@@ -295,7 +284,9 @@
   ;; get all option's values
   (let* (
 	 (report-title (get-option gnc:pagename-general optname-report-title))
-	 (company-name (get-option gnc:pagename-general optname-party-name))
+	 (book (gnc-get-current-book))
+	 ;; Get company-name should probably be modularized and standardized across reports
+	 (company-name (gnc:company-info book gnc:*company-name*))
```
Comment (jralls) [inline gnucash/report/reports/standard/income-statement.scm:289]: Same as on the Balance Sheet,
```
(company-name (or (gnc:company-info book gnc:*company-name*) ""))
```
It passes the tests here but without this change the report title will be "#f Income Statement for..." if there's no company name set in book options.

**Comment by jralls** [comment]
@christopherlam WTF? Why are you pushing irrelevant typo fixes onto this PR?

**Pull request closed without merging**